view/frontend/web/js/amwal-magento-react-button/src/AmwalMagentoReactButton.tsx

```diff
@@ -56,8 +56,8 @@ const AmwalMagentoReactButton = ({
       .catch(err => { console.log(err) })
   }, [])
 
-  const getQuote = async (addressData?: IAddress): Promise<void> => {
-    const response = await fetch('/rest/V1/amwal/get-quote', {
+  const getQuote = (addressData?: IAddress): Promise<void> => {
```

Comment:
`getQuote` shouldn't be modified as it throws the error as intended. Instead the calling should handle the thrown exception which is exactly what was done in line 120